.github/workflows/publish-commit-bottles.yml

```
@@ -316,12 +318,18 @@ jobs:
             --workflows=tests.yml \
             --committer="$BREWTESTBOT_NAME_EMAIL" \
             --root-url="https://ghcr.io/v2/homebrew/core" \
+            --retain-bottle-dir \
             ${{inputs.warn_on_upload_failure && '--warn-on-upload-failure' || ''}} \
             ${{inputs.message && '--message="$INPUT_MESSAGE"' || ''}} \
             "$PR"
 
           echo "head_sha=$(git rev-parse HEAD)" >> "$GITHUB_OUTPUT"
 
+      - name: generate build provenance
+        uses: github-early-access/generate-build-provenance@main
+        with:
+          subject-path: '${{steps.pr-pull.outputs.bottle-path}}/*.tar.gz'
```

```suggestion
          subject-path: '${{steps.pr-pull.outputs.bottle-path}}/*.tar.gz'
```

This says `bottle-path` but pr-pull.rb has `bottle_path`.

---

Thank you for catching this! I wish I had some way to run this to catch mistakes like this. But you saved us a cycle of committing and reverting, so thank you 😅

---

`actionlint` may catch some of this stuff. we should consider using it on this repo IMO.

---

`actionlint` probably doesn't know much about what the Ruby side does